src/components/ebay-pagination/index.js

```diff
@@ -107,12 +107,22 @@ function handlePageClick(event) {
 
 function handleNextPage(event) {
     eventUtils.preventDefaultIfHijax(event, this.state.hijax);
-    emitAndFire(this, 'pagination-next', { el: event.target });
+    emitAndFire(this, 'pagination-next', { el: getCorrectElement(event) });
 }
 
 function handlePreviousPage(event) {
     eventUtils.preventDefaultIfHijax(event, this.state.hijax);
-    emitAndFire(this, 'pagination-previous', { el: event.target });
+    emitAndFire(this, 'pagination-previous', { el: getCorrectElement(event) });
+}
+
+/**
+ * gets the correct target for pagination arrow button
+ */
+function getCorrectElement(event) {
+    if (event.target.nodeName === 'SPAN') {
```

Comment:
@sharma46bhawana It is better to explicitly upper case it `event.target.toUpperCase() === 'SPAN'`, to avoid weird [browser issues](https://johnresig.com/blog/nodename-case-sensitivity/).